src/index.js

```diff
@@ -1,9 +1,6 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-const Index = () => (
-    <div>
-      <h1>Hello! Welcome to Authors Haven</h1>
-    </div>
-);
-render(<Index/>, document.getElementById('root'));
```

Comment:
<img width="1440" alt="screen shot 2018-10-30 at 1 58 22 pm" src="https://user-images.githubusercontent.com/23438586/47772597-76978e00-dce7-11e8-853e-f5fd049197e6.png">

I moved the highlighted part of the code to a different file because of the above error